Resources/Prototypes/DeltaV/Entities/Mobs/Species/rodentia.yml

```diff
@@ -122,13 +122,13 @@
   - type: Flammable
     fireStackIncreaseMultiplier: 1.25
   - type: Inventory
-    speciesId: rodentia
-    femaleDisplacements: # Floof - displacements
-      jumpsuit:
-        sizeMaps:
-          32:
-            sprite: Mobs/Species/Human/displacement.rsi
-            state: jumpsuit-female
+    # speciesId: rodentia
```

Comment:
Please make sure not to comment out the speciesId fields, they may be needed for other purposes

Reply:
oops